aws/cloudfront_distribution_configuration_structure.go

```diff
@@ -266,54 +180,51 @@ func flattenCacheBehaviors(cbs *cloudfront.CacheBehaviors) []interface{} {
 	return lst
 }
 
-// Deprecated.
-func expandCacheBehaviorDeprecated(m map[string]interface{}) *cloudfront.CacheBehavior {
-	cb := &cloudfront.CacheBehavior{
+func expandCloudFrontDefaultCacheBehavior(m map[string]interface{}) *cloudfront.DefaultCacheBehavior {
+	dcb := &cloudfront.DefaultCacheBehavior{
 		Compress:               aws.Bool(m["compress"].(bool)),
-		FieldLevelEncryptionId: aws.String(m["field_level_encryption_id"].(string)),
-		ViewerProtocolPolicy:   aws.String(m["viewer_protocol_policy"].(string)),
-		TargetOriginId:         aws.String(m["target_origin_id"].(string)),
-		ForwardedValues:        expandForwardedValues(m["forwarded_values"].(*schema.Set).List()[0].(map[string]interface{})),
 		DefaultTTL:             aws.Int64(int64(m["default_ttl"].(int))),
+		FieldLevelEncryptionId: aws.String(m["field_level_encryption_id"].(string)),
+		ForwardedValues:        expandForwardedValues(m["forwarded_values"].([]interface{})[0].(map[string]interface{})),
 		MaxTTL:                 aws.Int64(int64(m["max_ttl"].(int))),
 		MinTTL:                 aws.Int64(int64(m["min_ttl"].(int))),
+		TargetOriginId:         aws.String(m["target_origin_id"].(string)),
+		ViewerProtocolPolicy:   aws.String(m["viewer_protocol_policy"].(string)),
 	}
 
 	if v, ok := m["trusted_signers"]; ok {
-		cb.TrustedSigners = expandTrustedSigners(v.([]interface{}))
+		dcb.TrustedSigners = expandTrustedSigners(v.([]interface{}))
 	} else {
-		cb.TrustedSigners = expandTrustedSigners([]interface{}{})
+		dcb.TrustedSigners = expandTrustedSigners([]interface{}{})
 	}
 
 	if v, ok := m["lambda_function_association"]; ok {
-		cb.LambdaFunctionAssociations = expandLambdaFunctionAssociations(v.(*schema.Set).List())
+		dcb.LambdaFunctionAssociations = expandLambdaFunctionAssociations(v.(*schema.Set).List())
 	}
 
 	if v, ok := m["smooth_streaming"]; ok {
-		cb.SmoothStreaming = aws.Bool(v.(bool))
+		dcb.SmoothStreaming = aws.Bool(v.(bool))
 	}
 	if v, ok := m["allowed_methods"]; ok {
-		cb.AllowedMethods = expandAllowedMethodsDeprecated(v.([]interface{}))
+		dcb.AllowedMethods = expandAllowedMethods(v.(*schema.Set))
 	}
 	if v, ok := m["cached_methods"]; ok {
-		cb.AllowedMethods.CachedMethods = expandCachedMethodsDeprecated(v.([]interface{}))
-	}
-	if v, ok := m["path_pattern"]; ok {
-		cb.PathPattern = aws.String(v.(string))
```

Comment:
[`PathPattern` is not a struct field in `DefaultCacheBehavior`](https://docs.aws.amazon.com/sdk-for-go/api/service/cloudfront/#DefaultCacheBehavior) so is removed here.